### ISSUE TITLE
feature request: links in Pango text

### ISSUE THREAD

**Issue by cedardocster** [v1cont/yad#102]
It would be super useful if this existed in Yad:

In a --formatted section (where Pango formatting is used) make a specific format similar like [[pretty link text|actual link text]] whereby the script tenders the pretty link text in a link format (e.g. blue underlined) and the actual link text when clicked is treated as a double click command, like "bash -c 'command here' " or "bash -c 'command here %s' " (not sure how that %s would work)

So ideally when clicked the script would run the function specified on the right side with the parameter (text) specified on the left side

E.g. [[some file name|gotolink]] would send that text "some file name" to the function named "gotolink"

Thank you.

**Comment by v1cont** [comment]
i think html dialog with --print-uri or --uri-handler options is more appropriate choice for such kind of tasks

**Comment by Botspot** [comment]
I agree with the OP. Such a feature would be very beneficial for a situation like this:  
![2020-12-16-120839_1920x1080_scrot](https://user-images.githubusercontent.com/54716352/102388515-7e821d00-3f97-11eb-8f82-86b58e0308a5.png)
It's a list with an important message at the top. It would be very nice if that link was clickable. (Or at least selectable)
